woocommerce/woocommerce-gateway-stripe
# fix: UPE banner eligibility

# Changes proposed in this Pull Request:
The UPE banner should not be displayed on Multibanco (since it's never going to be supported in UPE) or Alipay (since it's not yet eligible for UPE).

# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe_alipay or http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe_multibanco
- Banner is no longer displayed
